InfiniteLoopLMSSQL/src/com/st/lmssql/utils/DateCalculations.java

```diff
@@ -0,0 +1,61 @@
+package com.st.lmssql.utils;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class DateCalculations {
+
+	//gets today's date in yyyy-MM-dd
+	public static Date getCurrentTime() {

```

Comment:
As we have been instructed, don't use `java.util.Date` or `java.util.Calendar` unless you absolutely need to (and if you *do* need to, there should be an obvious comment explaining why).

Reply:
You can use LocalDate and LocalDateTime.